src/pages/EntityDetails/_entity-details.scss

```diff
@@ -294,45 +289,95 @@
     }
   }
 
-  .entity-details__units.has-checkbox {
-    th,
-    td {
-      &:nth-child(1) {
-        width: 4%;
-      }
+  .entity-details__units {
+    &:not(.has-checkbox) {
+      th,
+      td {
+        // Agent.
+        &:nth-child(3) {
+          width: 10%;
+        }
 
-      &:nth-child(2) {
-        width: 20%;
-      }
+        // Machine.
+        &:nth-child(4) {
+          width: 15%;
+        }
 
-      &:nth-child(3) {
-        width: 15%;
-      }
+        @include mobile-and-medium {
+          // Agent.
+          &:nth-child(3) {
+            width: 15%;
+          }
+
+          // Machine.
+          &:nth-child(4),
+            // Message
+            &:nth-child(7),
+            // Ports.
+            &:nth-child(6) {
```

Comment:
I think the indentation might be slightly off here.

Reply:
Fixed.